### PR TITLE
IDE-2137 update host bundle when install and uninstall fragment

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundleSupervisor.java [hunk 1]
```diff
@@ -130,4 +150,26 @@ public boolean stdout( String out ) throws Exception
...
+                fragmentHostId = bundle.id;
+                break;
+            }
+        }
+
+        if( fragmentHostId > 0 )
+        {
+            Agent agent = getAgent();
+            agent.redirect( Agent.COMMAND_SESSION );
+            agent.stdin( "update " + fragmentHostId );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundleSupervisor.java]: Hey @AndyWu2015  why can't you use the ```agent.update(...)``` api instead of having to pass this string?

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java [hunk 2]
```diff
@@ -120,7 +120,7 @@ private File checkStaticServicesFile() throws IOException
                 }
             }
 
-            if( !supervisor.getAgent().redirect( -1 ) )
+            if( !supervisor.getAgent().redirect( Agent.COMMAND_SESSION ) )
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: This service command seems to be unrelated to this issue.  If this is a bug can you submit it in another ticket where we discuss the problems in new JIRA?  thanks!

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java [hunk 3]
```diff
@@ -32,6 +32,7 @@
 import com.liferay.ide.server.remote.IServerManagerConnection;
 
 import java.io.File;
+import java.io.FileInputStream;
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java]: Did you format extra lines in this file?  Please restore them so it is easier for me to see what you changed here.  Thanks.

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java (hunk 2 above)
Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: yeah , Greg , I just replace the int var with static var which is better to understand.

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundleSupervisor.java (hunk 1 above)
Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundleSupervisor.java]: the update method need sha of the bundle , but I didn't find a way to get it on ide side .

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java (hunk 3 above)
Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java]: yeah , I will leave the original code right there without formatitng.

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java (hunk 2 above)
Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java]: the real value are the same.
